app/actions/views/command.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {Alert} from 'react-native';
+
+import {executeCommand as executeCommandService} from 'mattermost-redux/actions/integrations';
+import {getCurrentChannelId} from 'mattermost-redux/selectors/entities/channels';
+import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
+
+export function executeCommand(message) {
+    return async (dispatch, getState) => {
+        const state = getState();
```

Comment:
In another RN app I have seen this LOC (ie putting the entire state into a variable) be slow. In the case I'm referring to it only manifested *on an actual device* rather than on the emulator. Perhaps it's worth a quick comparison of doing this vs passing in `channelId` and `teamId` as params to the `executeCommand` function and then deciding if it's worth the change.

Reply:
Just measured this, it took 0.01ms to for `getState()` to occur so I don't think that justifies not using it. Glad you pointed that out though, I've changed it to where only one of the selectors runs by passing in `channelId` since we have that already in the post textbox.